Dockerfile

```diff
@@ -1,5 +1,5 @@
 # debian:buster 2021-12-20
-FROM debian@sha256:94ccfd1c5115a6903cbb415f043a0b04e307be3f37b768cf6d6d3edff0021da3
+FROM debian:bookworm
```

Comment:
We should figure out whether we want to keep building server kernels on buster or if we can do it on bookworm too.

Reply:
I'm going to punt on that for a separate PR.